connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -105,8 +106,8 @@
             "indicates that a configuration value will expire in the future.";
 
     private static final String CONFIG_RELOAD_ACTION_DISPLAY = "Reload Action";
-    public static final String CONFIG_RELOAD_ACTION_NONE = Herder.ConfigReloadAction.NONE.toString();
-    public static final String CONFIG_RELOAD_ACTION_RESTART = Herder.ConfigReloadAction.RESTART.toString();
+    public static final String CONFIG_RELOAD_ACTION_NONE = Herder.ConfigReloadAction.NONE.name().toLowerCase(Locale.ROOT);
```

Comment:
Any reasons we would want to explicitly make these constant values to be lower case?  I see that we only compare enums and we are doing an explicit upper case conversion there. Wouldn't that be good enough?

Reply:
The docs were out of sync since they referred to using lower case in the props file.  This change makes the validator to be in sync with the docs so that lower case can be used in the props file.